app/controllers/admin/slides_controller.rb

```diff
@@ -1,24 +1,26 @@
-class Admin::SlidesController < Admin::ApplicationController
-  def index
-    ransack_params = params[:q]
-    @q = Slide.search(ransack_params)
-    @slides = @q.result(distinct: true).
-              latest.
-              paginate(page: params[:page], per_page: 20)
-  end
+module Admin
+  class SlidesController < Admin::BaseController
```

Comment:
__[reek]__
* [InstanceVariableAssumption] assumes too much for instance variable @q [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)
* [InstanceVariableAssumption] assumes too much for instance variable @slide [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)

<!-- Comment by [SideCI](https://sideci.com) -->